packages/paste-website/src/pages/illustrations/index.mdx

```diff
@@ -0,0 +1,85 @@
+  {
+    mdx(fields: {slug: {eq: "/illustrations/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+<h1>{props.pageContext.frontmatter.title}</h1>
+
+<p>{props.pageContext.frontmatter.description}</p>
+
+---
+
+Illustration helps breathe life into our brand. It can take a complex idea and make it easier to understand. It is a tool that allows us to communicate our product in a way that feels human and fun.
+
+Our illustration style is carefully curated by the Twilio Brand team. While adding these elements to your UI, ensure that you are reviewing your work with them. A good place to start is by posting in #mkg-brand.
+
+<Box as="div" marginBottom="space60"><Stack orientation="vertical" spacing="space60">
+<Flex display="inline-flex" vAlignContent="center">
+<Anchor href="https://www.twilio.com/brand/guidelines/illustration-guidelines">
+Read the brand illustration guidelines
+</Anchor>
+<LinkExternalIcon decorative={false} title="Opens an external resource" />
+</Flex>
+
+<Flex display="inline-flex" vAlignContent="center">
+<Anchor href="https://library.twilio.com/web/2f44a10496e4c83b/shared-illustrations/">
+Browse the illustration repository
+</Anchor>
+<LinkExternalIcon decorative={false} title="Opens an external resource" />
+</Flex>
+</Stack></Box>
+
+### General guidelines for product UIs
+
+- Only make necessary and thoughtful adjustments to the illustration colors. When doing so, ensure that you&rsuo;re using Paste's palette.
+- Do not add extraneous elements to the illustrations. You may remove unnecessary elements, but do so while maintaining the thematic feel that the illustration is trying to convey.
+- We recommend using SVGs over PNGs when possible, in order to improve performance and avoid blurring.
+
+### Accessibility
+
+- **Aim to pair your illustrations with text:** Remember that different cultures may interpret the same image in different ways so despite our best intentions it is helpful to provide adjoining text.
+- **Illustrations shouldn't be interactive**: Illustrations should not have interactions such as click or hover behavior. Instead they should be wrapped with an (or, ideally, laid out with supporting) interactive element such as Anchor.
+- **Give your illustrations a title:** A title is required so assistive technology can infer equal meaning as a sighted user would.
+
+### Request new illustrations
+
+<Paragraph>
+  The best way to request new illustrations is to file a request with the Brand team. You can create a new{' '}
+  <Flex display="inline-flex" as="span" vAlignContent="center">
+    <Anchor href="https://issues.corp.twilio.com/projects/MARK/issues/"> request on JIRA</Anchor>
+    <LinkExternalIcon display="inline-block" decorative={false} title="Opens an external resource" /> (VPN required)
```

Comment:
Alignment and icon sizing gets wonky on all the anchors + icons when the browser shrinks in width.